### DISCUSSION TITLE
Support to export Tolgee tags / Have grouped translations

### DISCUSSION THREAD

**Discussion by MetalArend** [tolgee/tolgee-platform#1409] [Ideas]
Started playing around with Tolgee a few days ago, but so far I could not find a way to get the tags exported in the translations files. It's strange to see that it's not possible to export all data, and completely import them into another project.

The real thing we were looking for, is to have multiple groups of translations in a project. Tags seemed to be a possibility, but they are not exported, nor do the seem to be supported in the REST API. As for having groups, it's basically to have bigger parts of translations in one big project, and have some interface to reflect that bigger part you're in. Eventually, also use scope restrictions on them. Is that something that is planned? That might become possible in the future? Or did I miss something in the documentation?

**Comment by JanCizmar** [comment]
Hi there! We are currently working on that! It's called namespaces.

https://github.com/tolgee/tolgee-platform/issues/1359

The backend is ready, we are currently working on support in our JS integrations and for implementing it on frontend.
